### PR TITLE
Adds new ammo boxes, adds them to cargo and granularizes ammo box cargo prices.

### PR DESCRIPTION
**Pull request by Martinpachu**
## About The Pull Request

As the name says.
## Why It's Good For The Game

More ammo boxes are good i think. Makes more types of ammo less incredibly expensive to use. Guns have already been granularized and as such i think doing the same for ammo boxes is appropriate.

## Changelog

:cl:
add: Ammo boxes for 4.6, 8x50, Ferromagnetic slugs, pellets, lances and 8x50 Hollow Points!
balance: Granularized ammo boxes in cargo in a similar way to guns.
/:cl:

### CODE REVIEW

**Comment by Martinpachu** [comment]
my most painful PR yet

File: code/modules/projectiles/boxes_magazines/external/smg.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "A compact, 30-round top-loading magazine for the WT-550 Automatic Rifle. These rounds do okay damage with average performance against armor."
 	icon_state = "46x30mmt-30"
 	base_icon_state = "46x30mmt"
-	ammo_type = /obj/item/ammo_casing/c46x30mm
+	ammo_type = "4.6x30mm"
```
Comment (SomeguyManperson) [inline code/modules/projectiles/boxes_magazines/external/smg.dm]: Probably unintended?

**Comment by BarteG44** [comment]
the only way to get .308 at the moment it via the outpost for 215 credits per bullet

**Comment by Martinpachu** [comment]
> the only way to get .308 at the moment it via the outpost for 215 credits per bullet

it's not 215 per bullet?

**Comment by BarteG44** [comment]
.357*

also it's 107 per bullet since i missed that there are two speedloaders per crate

**Pull request merged**